pkgs/by-name/po/poptracker/package.nix

```diff
@@ -0,0 +1,86 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, substituteAll
+, util-linux
+, SDL2
+, SDL2_ttf
+, SDL2_image
+, openssl
+, which
+, libsForQt5
+, makeWrapper
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "poptracker";
+  version = "0.25.7";
+
+  src = fetchFromGitHub {
+    owner = "black-sliver";
+    repo = "PopTracker";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-wP2d8cWNg80KUyw1xPQMriNRg3UyXgKaSoJ17U5vqCE=";
+    fetchSubmodules = true;
+  };
+
+  patches = [
+    (substituteAll {
+      src = ./assets-path.diff;
+      assets = "${placeholder "out"}/assets";
+    })
+  ];
+
+  enableParallelBuilding = true;
+
+  nativeBuildInputs = [
+    util-linux
+    makeWrapper
+  ];
+
+  buildInputs = [
+    SDL2
+    SDL2_ttf
+    SDL2_image
+    openssl
+  ];
+
+  buildFlags = [
+    "native"
+    "CONF=RELEASE"
+    "VERSION=v${finalAttrs.version}"
+  ];
+
+  doCheck = true;
+
+  checkFlags = [
+    "CONF=RELEASE"
+    "VERSION=v${finalAttrs.version}"
+  ];
```

Comment:
It looks like the checks try to write to `$HOME` which points to a nonexistent directory. if a writable directory is needed, typically people will do:
```nix
preCheck = ''
  export HOME="$(mktemp -d)"
'';
```

even with HOME pointing to a writable location it seems that the tests require network access, which does not exist in the build sandbox, so the http connections fail but the test still succeeds. though, even with a bogus home directory the tests succeeded -- i wonder if it is even possible to get a non-zero exit value running the tests.